handbook/communication/content_guidelines/voice_and_tone.md

```diff
@@ -0,0 +1,95 @@
+# Voice and tone
+
+One of the ways we create a consistent and familiar experience for our users is by speaking with a consistent voice throughout our product and content while being aware of our tone.
+
+## What’s the difference between voice and tone?
+
+What’s the difference between voice and tone? Think of it this way: our **voice** is a personification of who we are. We always have the same voice. But, our **tone** changes to match our context and situation, usually through word choice.
+
+## Voice guidelines
+
+Our brand has a distinct voice and character. This voice and character is the basis of our writing for all of our digital platforms.
+
+When we write in our shared voice, we are always:
+
+### (This section is todo)
+
+- (This section is todo)
+    
+## Using tone
+
+Our tone is more than just the words we choose: it’s the way that we choose to share our personality through our voice.
+
+We can express any given message in wildly different ways. To find the right tone for any given message, consider the four dimensions of tone:
+
+1. Funny vs. serious.
+2. Formal vs. casual.
+3. Respectful vs. irreverent.
+4. Enthusiastic vs. matter-of-fact.
+
+Vary these dimensions to meet the user’s emotional and situational context at a given moment in time. Note that our brand voice has a “range” along these dimensions, and we should aim to stay within that range to keep our voice and character consistent.
+
+**Further reading:**
+
+- [The Four Dimensions of Tone of Voice](https://www.nngroup.com/articles/tone-of-voice-dimensions/) by NNG
+- [Tone of Voice](https://monzo.com/tone-of-voice/) by Monzo
+
+## Adapting to emotional context 
+
+### Positive situations
```

Comment:
For all those 3 types of situations as well - examples help understand the meaning correctly.

Reply:
I'm capturing this as an action item for myself—it'll take me a bit to come up with good examples, and I'd love to do it once we have the brand personality better defined.